src/systems/detachable_joint/DetachableJoint.hh

```diff
@@ -48,7 +50,18 @@ namespace systems
   /// - `<child_link>`: Name of the link in the child model to be used in
   /// creating a fixed joint with a link in the parent model.
   ///
-  /// - `<topic>` (optional): Topic name to be used for detaching connections
+  /// - `<detach_topic>` (optional): Topic name to be used for detaching
```

Comment:
We should keep the `topic` parameter in the documentation since it's still valid and users might see it in a model and look here to find out what it means. We can say the use of `detach_topic` is preferred or encouraged.

Reply:
addressed [here](https://github.com/gazebosim/gz-sim/pull/1687/files#diff-087c0092b0bf11de46dfb2b8ce617f244227ec4eb93b1adfbc04229b9f0457c7R53-R54)